lib/index.js

```diff
@@ -38,20 +44,30 @@ module.exports = function (gulp, options) {
   }
 
   var lintOptions = options.lint;
-  if (!lintOptions.files) {
-    lintOptions.files = [
-      options.sourceFiles,
-      options.unitTestFiles,
-      options.integrationTestFiles,
-      'index.js',
-      'main.js',
-      'gulpfile.js',
-      'gulpfile.babel.js',
-      'config/**/*.json',
-      'lib/**/*.json',
-      'test/**/*.json'
-    ];
-  }
+  lintOptions.files = lintOptions.files || [
+    options.sourceFiles,
+    options.unitTestFiles,
+    options.integrationTestFiles,
+    'index.js',
+    'main.js',
+    'gulpfile.js',
+    'gulpfile.babel.js',
+    'config/**/*.json',
+    'config/**/*.js',
+    'lib/**/*.json',
+    'test/**/*.json'
```

Comment:
I don't exactly understand the reason for having a poorly formed JSON file as a fixture since the only thing you would be testing is the ability of node.js to fail loading poorly formed data or javascripts inability to parse that data. But this is a configurable option, these are just defaults that aid in the creation of a basic testing framework to make it easier to setup on the average package. If you want to avoid loading something, just configure the linted files option.